x/stake/keeper.go

```diff
@@ -41,6 +41,31 @@ func (k Keeper) InitGenesis(ctx sdk.Context, data json.RawMessage) error {
 	return nil
 }
 
+// get the current counter
```

Comment:
we should add more explanatory comment on what the counter is used for here

Reply:
Added